### PR TITLE
[Finishes #93637344] Renamed 'office' to 'org code'

### PR DESCRIPTION
**Pull request by phirefly**
Conflicts:
	app/views/ncr/work_orders/form.html.erb
	config/data/ncr.yaml

### CODE REVIEW

File: config/data/ncr.yaml [hunk 1]
```diff
@@ -204,279 +204,8 @@ BUILDING_NUMBERS:
...
-  - 'VA1545ZZ - HOFFMAN NO 2'
-  - 'VA1547ZZ - MANASSAS SHOP CENTER'
-  - 'VA1567ZZ - 929 S PICKETT ST'
-  - 'VA1573ZZ - KIMBERLY BLDG'
-  - 'VA1579ZZ - N-1'
-  - 'VA1592ZZ - 901-3 S PICKETT ST'
-  - 'VA1605ZZ - HERNDON WAREHOUSE'
-  - 'VA1628ZZ - EISENHOWER INDS BDG'
-  - 'VA1631ZZ - POTOMAC IND. CENTER'
-  - 'VA1644ZZ - FRANCONIA SHOP CTR'
```
Comment (afeld) [inline config/data/ncr.yaml]: Why were these removed?

**Comment by phirefly** [comment]
Thanks @afeld. Was likely lost in the merge. Updated!
